tools/provision

```diff
@@ -48,37 +46,31 @@ the Python version this command is executed with."""
     venv_dir = os.path.join(base_dir, venv_name)
     if not os.path.isdir(venv_dir):
         try:
-            return_code = subprocess.call(['virtualenv', '-p', options.python_interpreter, venv_dir])
+            return_code = subprocess.call([options.python_interpreter, '-m', 'venv', venv_dir])
         except OSError:
-            if subprocess.call(['which', 'virtualenv']):
-                print("{red}Please install the virtualenv package and try again.{end_format}"
-                      .format(red='\033[91m', end_format='\033[0m'))
-                sys.exit(1)
+            print("{red}Installation with venv failed."
```

Comment:
I think on Ubuntu, `python3-venv` is a separate package (but not sure, haven't checked). If so, this err message doesn't take into account of the fact that the library hasn't been installed yet.

Reply:
Yes, confirmed that it is a separate package: https://packages.ubuntu.com/search?keywords=python3-venv. Note that this is specific to Ubuntu only.

Reply:
Yes, I know that as well. So do I mention that in the error message or run a subprocess for that? I think we can just add an extra line for that.

Reply:
Yes, you should say in error message that if you are on Ubuntu, you haven't installed `python3-venv`.